Mathlib/Logic/Equiv/Basic.lean

```diff
@@ -1165,7 +1165,7 @@ def subtypeSubtypeEquivSubtypeExists (p : α → Prop) (q : Subtype p → Prop)
 
 /-- A subtype of a subtype is equivalent to the subtype of elements satisfying both predicates. -/
 @[simps!]
-def subtypeSubtypeEquivSubtypeInter (p q : α → Prop) :
+def subtypeSubtypeEquivSubtypeInter {α : Type u} (p q : α → Prop) :
```

Comment:
[#18543](https://github.com/leanprover-community/mathlib/pull/18543) contains a backport of the original change, as opposed to this PR which reverts the change.